Makefile

```diff
@@ -33,7 +33,7 @@ DESTDIR ?= $(shell $(COQBIN)/coqc -where)/../../
 DOCDIR ?= share/doc/coq-elpi/
 COQDOCINSTALL ?= $(DESTDIR)$(DOCDIR)/user-contrib
 
-export DESTDIR
+#export DESTDIR
```

Comment:
Ahah yes, srry, but I think this is not the right fix. I think the split between `DESTDIR` and `DOCDIR` was wrong...
Is it supposed to be
```
DESTDIR ?= $(shell $(COQBIN)/coqc -where)/../
DOCDIR ?= ../share/doc/coq-elpi/
```
maybe?
CC @gares